templates/cloud/index.html

```diff
@@ -505,35 +513,39 @@ <h2>MAAS：裸机即服务</h2>
         <li class="p-list__item is-ticked">部署Ubuntu、CentOS、Windows或自定义映像的最快方式</li>
         <li class="p-list__item is-ticked">REST风格的API、Web用户界面和命令行客户端</li>
       </ul>
-      <p><a href="https://www.ubuntu.com/server/maas">进一步了解MAAS&nbsp;›</a></p>
-    </div>
-    <div class="col-2 p-hero__item">
-      <img class="p-hero__image u-image-position--bottom" src="https://assets.ubuntu.com/v1/8ca2eab9-hero-ui-image.png" alt="MAAS UI web page showing node listing" style="max-height:100%;">
-    </div>
-  </div>
+
+      <ul class="p-list">
+      <li><img src="https://assets.ubuntu.com/v1/8ca2eab9-hero-ui-image.png" alt="MAAS UI web page showing node listing"></li>
+      </ul>
+      <p><a href="https://www.ubuntu.com/server/maas">进一步了解MAAS&nbsp;›</a></p>    
```

Comment:
This is an external link and therefore should have the external class an not the `&nbsp;>`